test/index.js

```diff
@@ -1,6 +1,23 @@
 const tape = require('tape')
 const bipf = require('../')
-const pkg = require('../package.json')
+
+const pkg = {
+  name: 'test',
+  version: '1.0.0',
+  description: 'test package',
+  repository: {
+    type: 'git',
+    url: 'git://github.com/ssbc/bipf.git',
+  },
+  dependencies: {
+    varint: '^5.0.0',
+  },
+  husky: {
+    hooks: {
+      'pre-commit': 'npm run format-code-staged',
+    },
+  },
+}
```

Comment:
Replaced the import of `package.json` to make these tests more deterministic (we don't want test behavior to change when we update package.json) and also to make it easier to test `bipf-native` using `bipf`'s test suite.